src/Sparrow/Utils/ZstdLib.cs

```diff
@@ -2,11 +2,13 @@
 using System.Diagnostics;
 using System.IO;
 using System.Runtime.InteropServices;
+using System.Security;
 using Sparrow.Platform;
 
 namespace Sparrow.Utils
 {
 #if NETCOREAPP3_1_OR_GREATER
+    [SuppressUnmanagedCodeSecurity]
```

Comment:
How much do we benefit from this?

Reply:
For reference: https://docs.microsoft.com/en-us/dotnet/api/system.security.suppressunmanagedcodesecurityattribute?view=net-6.0

Reply:
We hope a lot, need to profile again on the big machine